src/frontend/components/EmailModal/EmailModal.js

```diff
@@ -90,7 +90,7 @@ const EmailModal = ({ status, data, onSubmit, open, handleClose }) => {
     setBodyInput(text);
   }, [status, data]);
 
-  const execContact = `\n${execEmail} | ${execPhoneNum}`;
+  const execContact = `\n${execEmail}`;
```

Comment:
Minor: no need to address for this PR, but I think in the future we should use margins/padding to create a space.